libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -220,7 +220,17 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("SD_SLOWDOWN",  17,     AP_BoardConfig, _sdcard_slowdown,  0),
 #endif
-    
+
+#ifdef HAL_GPIO_PWM_VOLT_PIN
+    // @Param: PWM_VOLT_SEL
+    // @DisplayName: Set PWM Out Voltage
+    // @Description: This sets the voltage max for PWM output pulses. 0 for 3.3V and 1 for 5V output.
+    // @Range: 0 1
```

Comment:
this should be a @Values entry, not Range and Increment. We want a selection list, not a dial in UI

Reply:
Once you have a values entry you should probably remove the explicit list from the description.